services/ops/LogStatisticsAgent/README.md

```diff
@@ -0,0 +1,38 @@
+## Log Statistics Agent
+
+The Log Statistics agent periodically reads the "volttron.log" file, computes the size delta from the previous hour 
+and publishes the difference with a timestamp.  It also publishes standard deviation of the size delta every 24 hours.
+This agent is useful for ___.
+
+
+### Configuration
+
+The Log Statistics agent has 4 configuration values:
+
+- "file_path":  This should be the path to the "volttron.log" file, defaults to "$VOLTTRON_HOME/volttron.log"
+- "analysis_interval_secs":  The interval in seconds between publishing the size delta statistic to the message bus
+- "publish_topic":  Can be used to specify a topic to publish log statistics to which does not get captured by the 
```

Comment:
"Specifies a topic to which log statistics are published." What determines if the topic is included by the historian? could it be included? Is that based on the topic choice and the historian's config?

Reply:
This should be an issue, the log stats agent currently ignores many best practices and does not do what it says on the tin.